_posts/2022-06-23-reached-50000-users.md

```diff
@@ -0,0 +1,55 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Usegalaxy.eu has reached 50,000 registered users'
+tags: [galaxy, support, society, report]
+supporters:
+- denbi
+- elixir
+- dataplant
+- crc1425
+- crc992
+- biolytix
+- converge
+- bycovid
+- healthycloud
+- eosc
+- gallantries
+- pari
+- neic
+- miracum
+- biodaten
+- egi
+- corbel
+- bmbf
+- dfg
+- unifreiburg
+- galaxy-freiburg
+- galaxy-europe
+author: beatrizserrano
+---
+
+![50,000 users](/assets/media/50K_users/reached_50000_users.png)
+<br>
+
+The [European Galaxy server](https://usegalaxy.eu/) was [officially launched on __March 15, 2018__](https://galaxyproject.eu/posts/2018/03/15/usegalaxy-eu/) during the first [Galaxy User Conference](https://galaxyproject.eu/posts/2018/03/15/y-galaxy-user-conference-a-successful-first-day/) in Freiburg. Despite the short timespan, many things have changed since then: __from 800 users at that time till the 50,000 that we've reached this year!__ 
+
+The European Galaxy server provides __free computing resources to scientist all over the world__ (more than 100 countries!). The aim is to __support scientists__ at all career stages and __regardless of their socioeconomic background__, so that everybody gets the same chances to perform data analysis. Some of our users have shared with us some interesting [scientific uses cases](https://galaxyproject.eu/news?tag=UseCase) and [publications in scientific journals](https://galaxyproject.eu/citations) that show the importance of the use of the services in the scientific progress.
+
+Not only the amount of registered users have grown, the number of [tools installed](https://galaxyproject.eu/tools) is now over 2,800, 47M jobs have been run, 260K workflows have been executed... The training activities have also increased, reaching 13K trainees using [Training Infrastructure as a Service (TIaaS)](https://galaxyproject.eu/posts/2021/08/24/tiaas-flyer/) in [different aspects of science](https://galaxyproject.eu/news?tag=TIaaS). Take a look at these and other impressive numbers in our factsheet:
+
+<br>
+![factsheet](/assets/media/50K_users/factsheet_automatable_eu.png)
+<br><br>
+
+And we wanted to celebrate this important milestone with __all the people that have supported us__ in all sorts of ways: facility management, technical support from the compute center to manage an increasing [hardware infrastructure](https://galaxyproject.eu/news?tag=hardware), our sponsors, the EU office at the University of Freiburg, the Backofen lab, our power users, the [whole team](https://galaxyproject.eu/people) and the rest of the Galaxy community!
```

Comment:
compute center could be linked and the uni-freiburg as well, backofen lab ...

Reply:
add  Science Support center (Frank Krüger)?
Medical faculty?
MPI?

Mention support of Prorektor für Forschung ? @bgruening ?